qcelemental/models/molecule.py

```diff
@@ -131,7 +139,8 @@ def __init__(self, orient=False, validate=True, **kwargs):
         values = self.__values__
 
         natoms = values["geometry"].shape[0]
-        values["symbols"] = np.core.defchararray.title(self.symbols)  # Title case for consistency
+        if validate:
+            values["symbols"] = np.core.defchararray.title(self.symbols)  # Title case for consistency
```

Comment:
are the "symbols" and "geometry" re-definitions in order to satisfy the skip_defaults note above? I was going to suggest that `if kwargs["validated"]:` would make clearer reading than `if validate:` but I guess they're functionally equivalent.

So if one deliberately declines validation, `GEOMETRY_NOISE` and `defchararray` don't get applied? I can see the former, as decimal stripping is "chemical", but code relies on the titlecase of the symbols -- ok to skip that?

Reply:
`validated` is for shipping around Molecules that are already guaranteed to be passed through a Molecule class already. Ideally a user would never set `validated` themselves (or hopefully know about it).

As a note that `validate` and `validated` are different. `validate` can be overridden during init to force a validation on the input regardless of the `validated` kwarg.